packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -34,7 +34,7 @@ class CheckboxGroup extends React.Component {
             name={option.name}
             value={option.value}
             checked={includes(this.props.values, option.value)}
-            onChange={val => this.handleChange(option.value, val)}/>;
+            onChange={e => this.handleChange(option.value, e.target.checked)}/>;
```

Comment:
Meh, it's not going to buy us much until React Fiber lands. Before that, it's just a nicer syntax.